pkg/cosign/tlog.go

```diff
@@ -436,12 +450,7 @@ func VerifyTLogEntry(ctx context.Context, _ *client.Rekor, e *models.LogEntryAno
 		LogID:          *e.LogID,
 	}
 
-	rekorPubKeys, err := GetRekorPubs(ctx, nil)
-	if err != nil {
-		return fmt.Errorf("unable to fetch Rekor public keys: %w", err)
-	}
-
-	pubKey, ok := rekorPubKeys[payload.LogID]
+	pubKey, ok := rekorPubKeys.Keys[payload.LogID]
```

Comment:
I'm so sorry, I missed this last time around :(

We should check for rekorPubKeys for being nil now that it's inside the wrapper struct. We do it in VerifyBundle, but not here.

Reply:
Maybe right after checking that e.Verification is not nil? L425?

Reply:
Nice catch, thanks!